draft-ietf-dnsop-svcb-https.md

```diff
@@ -1082,18 +1082,31 @@ single-client granularity.
 
 ## Interaction with Alt-Svc
 
-Clients that do not implement support for Encrypted ClientHello MAY
-skip the HTTPS RR query
-if a usable Alt-Svc value is available in the local cache.
-If Alt-Svc connection fails, these clients SHOULD fall back to the HTTPS RR
-client connection procedure ({{client-behavior}}).
-
-Clients that implement support for ECH MUST perform the HTTPS RR query first,
-and MUST only make use of Alt-Svc when operating in SVCB-optional mode (see
-{{ech-client-behavior}}).
-
-This specification does not alter the DNS records used when connecting
-to an Alt-Svc hostname (typically A and/or AAAA only).
+Clients that implement support for both Alt-Svc and HTTPS records SHOULD
+retrieve any HTTPS records for the Alt-Svc alt-authority, and ensure that
+their connection attempts are consistent with both the Alt-Svc parameters
+and any received HTTPS SvcParams.  If present, the HTTPS record's target
+and port override the alt-authority.  For example, suppose
+"https://example.com" sends an Alt-Svc field value of:
+
+    Alt-Svc: h3="alt.example.com:8443"; ...
+
+The client would retrieve the following HTTPS record:
+
+    _8443._https.alt.example.com. ... 1 alt2.example.com. (
```

Comment:
Do we also want to add an example in the appendix without the port prefix?

Reply:
It seems clear enough to me as-is, but if you think it's confusing, we can add it.

Reply:
It seems good to add just to be crystal clear that h3="alt.example.com:443" only looks up the HTTPS RR for "alt.example.com".
Just showing both forms here as below might be one way to do this.

Reply:
OK, I added your suggested example and tweaked it slightly to (1) simplify, (2) cover the case where there is no `port` SvcParam, and (3) cover the case where the Alt-Svc hostname is implicit.